Description: Install header files into include, not in include/<subfolder>
Forwarded: no
Author: arnebe <arne@alamut.de> 

--- a/cmake_modules/SetupDirectories.cmake
+++ b/cmake_modules/SetupDirectories.cmake
@@ -9,7 +9,8 @@
 
 # Installation prefix for include files
 STRING (TOLOWER ${PROJECT_NAME} projectNameLower)
-SET (PROJECT_INCLUDE_INSTALL_DIR "include/${projectNameLower}")
+#SET (PROJECT_INCLUDE_INSTALL_DIR "include/${projectNameLower}")
+SET (PROJECT_INCLUDE_INSTALL_DIR "include")
 
 IF (NOT DEFINED PROJECT_LIBRARY_INSTALL_DIR)
 	SET (PROJECT_LIBRARY_INSTALL_DIR "lib" CACHE PATH "object code libraries")
